Added request invoice button & staff pricing

mihairadulescucom opened the pull request.

lucastobrazil commented:
@mihairadulescucom all looking good! just two things
1. The css !important thing i commented on
2. there's a few console.logs in the javascript(s) - are you able to remove them please?

it might have been me that put them 👅 

i can merge after that

mihairadulescucom commented:
@lucastobrazil 
1. Couldn't see any comments, maybe I'm missing something on here. Removed all !important rules.
2. Could only find one console.log; removed it.

Let me know if all good.
